src/lib/components/PreTestTimer.svelte

```
@@ -0,0 +1,94 @@
+<script lang="ts">
+	import { enhance } from '$app/forms';
+	import { page } from '$app/state';
+	import { Button } from '$lib/components/ui/button';
+	import * as Dialog from '$lib/components/ui/dialog';
+
+	const startTime = new Date(page.data.testData.start_time);
+	const startTimeString = startTime.toString().split(' GMT')[0];
+
+	const { timeLeft: initialTime } = $props();
```

ESLint

[error] 'initialTime' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)